kloppy/infra/serializers/tracking/tracab.py

```diff
@@ -154,6 +158,9 @@ def deserialize(
         sample_rate = float(options.get("sample_rate", 1.0))
         limit = int(options.get("limit", 0))
         only_alive = bool(options.get("only_alive", True))
+        to_coordinate_system = options.get(
```

Comment:
Aha.. now I see what you mean. I would pass a class instead of an instance.
`to_coordinate_system=KloppyCoordinateSystem`

Reply:
@koenvo I like the idea of passing a name! May be even better if the builder takes in the `Provider` Enum?

Reply:
Uhm yes. Accept the exact value of the `Provider` enum. But lets not bother the end-user with passing the `Provider.TRACAB` value itself.